jenkins/build.sh

```diff
@@ -76,6 +120,7 @@ echo "Configuring the build with: $CONFIGURE_FLAGS"
 # shellcheck disable=SC2086
 ./configure $CONFIGURE_FLAGS
 
+sleep 3600 # test timeout support
```

Comment:
I've modified Jenkins to pass --timeout=3600 to build.sh, this line is to test that it times out as expected. It will be removed before merging the PR.